components/CompositeCharacterImage/traits.ts

```diff
@@ -38,6 +38,17 @@ export type TraitsArray = [
   string,
 ];
 
+export const DEFAULT_TRAITS: TraitsArray = [
```

Comment:
Only thing I might change is move this file to under `lib` or `utils` but that's personal preference.

Reply:
Addressed in https://github.com/raid-guild/character-sheets-frontend/pull/122